ktcacheorbust.cc

```diff
@@ -154,23 +184,27 @@ bool CacheOrBust::Worker::do_get(
 
   if (tokens.size() < 2)
     return sess->printf("CLIENT_ERROR missing key\r\n");
-  if (tokens.size() < 3)
-    return sess->printf("CLIENT_ERROR missing URL\r\n");
   if (tokens.size() > 4)
     return sess->printf("CLIENT_ERROR extra data after TTL\r\n");
 
   std::string key(tokens[1]);
-  std::string url(tokens[2]);
-  if (tokens.size() == 4)
-    ttl = kc::atoi(tokens[3].c_str());
+
+  if (_serv->_log_keys) {
+    _serv->log(kt::ThreadedServer::Logger::INFO, "COB key request: %s", key.c_str());
+  }
+
+  if (! _serv->_strip_prefix.empty() && ! key.compare(0, _serv->_strip_prefix.length(), _serv->_strip_prefix)) {
+    // Strip leading prefix from key (for mcrouter prefix-routing)
```

Comment:
can this comment be expanded to show both an example of what it takes in and acts on, as well as a link to the relevant bit of documentation from mcrouter side?

Reply:
Sure